webview/src/shared/components/codeSlider/CodeSlider.tsx

```diff
@@ -0,0 +1,48 @@
+import cx from 'classnames'
+import React, { useState } from 'react'
+import styles from './styles.module.scss'
+import { CodeBlock, CodeBlockProps } from '../codeBlock/CodeBlock'
+
+interface CodeBlockWithTitle extends CodeBlockProps {
+  title: string
+}
+
+interface CodeSliderProps {
+  codeBlocks: CodeBlockWithTitle[]
+}
+
+export const CodeSlider: React.FC<CodeSliderProps> = ({ codeBlocks }) => {
```

Comment:
Is there an issue with using [`VSCodePanels`](https://github.com/microsoft/vscode-webview-ui-toolkit/blob/main/src/panels/README.md)? I'd assume we'd want to use them to make our web views as "vscode-like" as possible.

Reply:
Would probably resolve the comment about tabs improvements as well.

Reply:
Didn't know about `VSCodePanels` will try it.

Reply:
Done